x-pack/plugins/session_view/public/components/process_tree/hooks.ts

```diff
@@ -94,11 +94,6 @@ export class ProcessImpl implements Process {
           return false;
         }
 
-        // If the process has no children and has not exec'd (fork only), we hide it.
-        if (child.children.length === 0 && !child.hasExec()) {
```

Comment:
removing this as it was added because of a bug in endpoint where entity_id would change btw the fork and exec calls thus creating a bunch of lone forks.